DocsAndHospitals/Persistence/HospitalRepository.cs

```diff
@@ -0,0 +1,26 @@
+﻿using DocsAndHospitals.Models;
+using System.Text.Json;
+
+namespace DocsAndHospitals.Persistence;
+
+public static class HospitalRepository
+{
+    private static readonly string FilePath = "hospitals.json";
+
+    public static void Save(IEnumerable<Hospital> hospitals)
+    {
+        var options = new JsonSerializerOptions { WriteIndented = true };
```

Comment:
you create it every time and allocate memory - you could do it private static readonly field